source/using-web-interface.rst

```diff
@@ -244,4 +244,39 @@ After you create the template with the Firmware management option enabled, you c
 Note that in each step, the status and result of image processing are shown.
 
 
+Generating certificates for devices
+------------------
+
+This section will show how to generate x509 certificates for a device, so that dojot can communicate with devices securely via TLS. To be able to send a publication, it is necessary to download the three files, the private key "admin 4302d4.key", the device certificate "admin 4302d4.crt" and the CA certificate "ca.crt".
+
+To be able to execute the command of the example video it is necessary to be in the same directory as the 3 files attached. Command used in the example:
+
+.. code:: bash
+
+   mosquitto_pub -h 10.50.11.192 -p 8883 -t admin: 4302d4/attrs -m '{"humidity": 7}' --cert "admin 4302d4.crt" --key "admin 4302d4.key" --cafile ca.crt
```

Comment:
For anyone reading, this IP is meaningless, unless you explain that the IP is that of the dojot host...
could this IP be the loopback interface (`127.0.0.1`)?

Reply:
I switched to localhost